data/init.sql

```diff
@@ -195,3 +195,21 @@ CREATE TABLE IF NOT EXISTS announcements (
 	created_at TIMESTAMP,
 	updated_at TIMESTAMP
 );
+
+CREATE TABLE IF NOT EXISTS teams (
+	id INT(11) UNSIGNED AUTO_INCREMENT PRIMARY KEY,
+	course_offering_id INT(11) UNSIGNED,
+	FOREIGN KEY(course_offering_id) REFERENCES course_offerings(id) ON DELETE CASCADE,
```

Comment:
this now needs to change to lecture section or tutorial section (forgot which one we landed on) after zach's refactor is merged right?

Reply:
yeah probably we should merge that PR asap

Reply:
Yeah goint to do that now, I did it thisway so I could test it and make sure it worked

Reply:
done